egs/wham/ConvTasNet/local/prepare_data.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-wav_dir=tmp
-out_dir=tmp
+wav_dir=/mnt/data/wsj0-mix/wsj0
+out_dir=/mnt/data/wham
```

Comment:
I don't understand these lines, sorry. 
These arguments are supposed to be parsed from `run.sh` directly, these are just default values, we can keep the `tmp` as before.

Reply:
even tmp will not work for the wavdir I just puthem there for a new user to more easily understand where to look